pkg/arvo/sur/contact-store.hoon

```diff
@@ -1,10 +1,26 @@
 /-  *identity
 |%
-+$  rolodex  (map path contacts)
++$  rolodex-0  (map path contacts-0)
```

Comment:
Currently, we never expose older-version data structures in `/sur` files, instead always containing them in the `/app`, sometimes even in _just_ the `+on-load`. Is there a reason these need to be public, or can we hide them away?

Reply:
Good point.